test/rules/ordered-imports/groups-shared-order/test.ts.lint

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+import {f} from 'app_b/foo';
+import {x} from 'app_a/foo';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources within a group must be alphabetized.]
+import {y} from 'app_a/bar';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources within a group must be alphabetized.]
+
+
+import {a} from '@pkg/foo';
+
+import {b} from '@pkg/bar';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources of different groups must be sorted by: ^app_b, ^app_a, ^@pkg .]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import sources with same group order must be grouped together.]
```

Comment:
Took me a while to figure it out too and maybe the logic should/could be changed to make this one error.  The issue is that the rule is finding two problems here:  First, the import isn't with the other imports that have the same prefix.  (that is the error about same group order must be grouped together.  Second, if the group was together it would be out of order because this new group has the same order as the previous one.  

Long story short, I think this should probably be just the error about groups must be grouped together.  I will see if I can refactor the logic to make that happen.

Reply:
@adidahiya Addressed with the change to remove the "grouped together" check.